reasoner_util/__init__.py

```diff
@@ -1,12 +1,32 @@
 """Build merging functions."""
 from typing import Hashable, Iterable, List, TypeVar
+import httpx
 
 T = TypeVar("T", bound=Hashable)
 
+
 def merge_iterables(cats1: Iterable[T], cats2: Iterable[T]) -> List[T]:
     """Merge iterables."""
     return list(set(cats1) | set(cats2))
 
 
 merge_ids = merge_iterables
 merge_categories = merge_iterables
+
+
+def get_preferred_ids(curies: List[str]):
+    """Get list of preferred ids for a list of CURIES."""
+    params = {"curie": curies}
+    r = httpx.get("https://nodenormalization-sri-dev.renci.org/1.1/ \
+        get_normalized_nodes", params=params)
+    r_dict = r.json()
```

Comment:
```suggestion
    response_dict = response.json()
```

Reply:
@patrickkwang I updated the variable names to be more descriptive, but now I have a formatting issue with line 25, as it is now too long. Any suggestions?